chia/cmds/wallet_funcs.py

```diff
@@ -33,6 +34,44 @@ def print_transaction(tx: TransactionRecord, verbose: bool, name) -> None:
         print("")
 
 
+def get_csv_line(tx: TransactionRecord, name, date_fmt, delim):
+
+    chia_amount = Decimal(int(tx.amount)) / units["chia"]
+    fee_amount = Decimal(int(tx.fee_amount)) / units["chia"]
+    to_address = encode_puzzle_hash(tx.to_puzzle_hash, name)
+    tx_type = get_tx_type_as_string(tx.type)
+    tx_datetime = datetime.fromtimestamp(tx.created_at_time).strftime(date_fmt)
+    status = "Confirmed" if tx.confirmed else ("In mempool" if tx.is_in_mempool() else "Pending")
+
+    line_elems = [
+        tx.wallet_id,
+        tx_datetime,
+        tx_type,
+        status,
+        tx.confirmed_at_height,
+        tx.name,
+        chia_amount,
+        fee_amount,
+        to_address,
+    ]
+
+    return delim.join(map(str, line_elems))
+
+
+def get_tx_type_as_string(tx_type: int):
+    TX_TYPE_DICT = {
+        TransactionType.FEE_REWARD: "FEE_REWARD",
+        TransactionType.COINBASE_REWARD: "COINBASE_REWARD",
+        TransactionType.INCOMING_TX: "INCOMING_TX",
+        TransactionType.OUTGOING_TX: "OUTGOING_TX",
+        TransactionType.INCOMING_TRADE: "INCOMING_TRADE",
+        TransactionType.OUTGOING_TRADE: "OUTGOING_TRADE",
+        999: "UNKNOWN",
```

Comment:
I don't know if this code already exists somewhere, but fyi it's not necessary to be in the dict for `.get` to work.

Reply:
For some reason, mypy lint is failing without the UNKNOWN in the dict

Reply:
I don't imagine it's because of the change.
E.g. test 
```python
d={"a":2,"b":3}; d.get("c",4)
```
in a REPL/command line, there should be no issue with it.

Reply:
The code works properly without the UNKNOWN in the dict when I run it...  it just fails the mypy Lint.  Do I need to pass all the tests for the pull request to be accepted?

Reply:
I'd imagine so (I don't have a team tag or anything). 
But it's worth finding out what happens here in this case anyhow.

Reply:
When I remove the UNKNOWN from the dict I get this from mypy lint:
```bash
mypy wallet_funcs.py 
wallet_funcs.py:71: error: No overload variant of "get" of "Mapping" matches argument types "int", "str"
wallet_funcs.py:71: note: Possible overload variant:
wallet_funcs.py:71: note:     def [_T] get(self, key: TransactionType, default: Union[str, _T]) -> Union[str, _T]
wallet_funcs.py:71: note:     <1 more non-matching overload not shown>
Found 1 error in 1 file (checked 1 source file)
```

Reply:
Okay interesting, it seems that the tool has issues with the container type and is thus too strict, see
https://stackoverflow.com/questions/62574831/why-do-i-get-mypy-no-overload-variant-of-zip-matches-argument-error

Something else you can do, without the 999 hack, is 
```python
for key, val in TX_TYPE_DICT.items():
    if tx_type == key:
        return val
return "UNKNOWN"
```

Reply:
I figured it out...

```python

TX_TYPE_DICT = {
        int(TransactionType.FEE_REWARD): "FEE_REWARD",
        int(TransactionType.COINBASE_REWARD): "COINBASE_REWARD",
        int(TransactionType.INCOMING_TX): "INCOMING_TX",
        int(TransactionType.OUTGOING_TX): "OUTGOING_TX",
        int(TransactionType.INCOMING_TRADE): "INCOMING_TRADE",
        int(TransactionType.OUTGOING_TRADE): "OUTGOING_TRADE",
    }
```

I needed to cast them as int.

Reply:
Okay, so worth remembering the type lib is not the biggest friend of the `Enum` module, even as far as the members of standard lib function.

Here's a minimal example to show that raw python itself takes issue with it:
```python
>>> from enum import Enum
>>> class Animal(IntEnum):
...     DOG = 1
...     CAT = 2
... 
>>>
>>> int(Animal.DOG)
1
>>> type(Animal.DOG)
<enum 'Animal'>
>>>
>>> d = {Animal.DOG : 9001}
>>> d[1]
9001
>>> d.get(1)
9001
```